services/QuillLMS/client/app/bundles/Teacher/components/assignment_flow/create_unit/custom_activity_pack/index.tsx

```diff
@@ -214,7 +214,7 @@ const CustomActivityPack = ({
     setGradeLevelFilters(newGradeLevelFilters)
   }
 
-  function handleELLFilterChange(newELLFilters: string[]) {
+  function handleELLFilterChange(newELLFilters: number[]) {
```

Comment:
Is this change supposed to be in here?

Reply:
Good catch, Emilia. Cissy, I made this change a few days ago. You probably want to rebase against develop and check that the type is `string[]`